Storage/src/Bucket.php

```diff
@@ -157,12 +157,15 @@ public function defaultAcl()
      * }
      * ```
      *
+     * @param array $options [optional] {
+     *     Configuration options.
```

Comment:
List what the valid configuration options are?

Reply:
This matches the style of the rest of the library, so it looks correct based on the scope of this PR.

Reply:
Yeah, I left it intentionally because the only accepted options here are the ones which we pass to connection layer like `restOptions` etc, and these aren't documented in other methods.